app/components/util/ranking.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+
+export default function Ranking(rankingProps) {
+  const responseOptions = ['Most', 'Neutral', 'Least']
+  const displayPropOptions = ['block', 'small', 'medium', 'large']
+
+  //Define new and inherited classes
+  let classes = ['radial_ranking', ...(className ? className : [])]
+
+  //Isolate props and set initial state
+  const { disabled, defaultValue, className, onSelected, ...otherProps } = rankingProps
+  let [response, setResponse] = useState(responseOptions.includes(defaultValue) ? defaultValue : '')
+
+  displayPropOptions.forEach(prop => {
+    if (otherProps[prop]) {
+      classes.push(prop)
+      delete otherProps[prop]
+    }
+  })
+
+  const onSelectionChange = e => {
+    if (e.target.disabled) {
+      return
+    }
+    setResponse(e.target.value)
+    if (!onSelected) {
+      return console.warn(
+        `Unhandled rank selection: ${e.target.value}. Please pass a handler function via the onSelected prop.`
+      )
+    }
+
+    onSelected(e)
+  }
+
+  return (
+    <div data-value={response} className={classes.join(' ')} onChange={onSelectionChange} {...otherProps}>
```

Comment:
So I learned something. I've never used data-value before, I had to look it up. 😄